Gemfile

```diff
@@ -68,6 +68,7 @@ end
 group :test do
   # Use system testing [https://guides.rubyonrails.org/testing.html#system-testing]
   gem 'capybara'
+  gem 'rails-controller-testing'
   gem 'selenium-webdriver'
   gem 'webdrivers'
 end
```

Comment:
### Although I am highlighting this file, the change I am about to suggest is not related to this file. 🙏 

- [ ] When I run your migration files, I get this error:

![image](https://github.com/mercymugambi/blog-app/assets/77106962/ea3bd64f-d252-40e3-bc55-374ed5a8a954)

#### In the `20230717152751_remove_posts_count_from_users.rb` migration file, you are attempting to drop a column named `"posts_count"` that doesn't exist on the `Users` table. I would advise you to delete that migration file because the `Users` table is setup properly. 👍